SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/Component.kt

```diff
@@ -0,0 +1,38 @@
+package app.cybrid.sdkandroid.components
+
+import android.content.Context
+import android.os.Handler
+import android.os.Looper
+import android.util.AttributeSet
+import androidx.compose.ui.platform.ComposeView
+import androidx.constraintlayout.widget.ConstraintLayout
+
+open class Component @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = 0
+) : ConstraintLayout(context, attrs, defStyle) {
+
+    var updateInterval = 5000L
+
+    private var _handler: Handler? = null
+    private var _runnable: Runnable? = null
+    protected var composeView:ComposeView? = null
+    var isUpdating = false
+
+    fun setupRunnable(runner: () -> Unit) {
+
+        _handler = Handler(Looper.getMainLooper())
+        _runnable = Runnable {
+
+            isUpdating = true
+            runner.invoke()
+            _handler.let {
+                _runnable.let { _it ->
+                    it?.postDelayed(_it!!, updateInterval)
```

Comment:
Is Force Unwrap safe in Kotlin? It's considered a bad practice in Swift.

Reply:
Maybe, but kotlin still in baby steps, if you check `_runnable` run `let` that safety check if the object its not null and then referecne this not null as `_it` so this never is null, but in case its null, the runnable dont gonna work!!

Reply:
I added empty runnable in case this runnable for some weird reasn could be null, the runnable gonna make nothing

Reply:
I mean a simple `if (_it == null) return` could work as a safeguard.